packages/orbit-components/src/Loading/index.tsx

```diff
@@ -200,15 +79,38 @@ const Loading = ({
 }: Props) => {
   return (
     <>
-      {children && !loading ? (
+      {Boolean(children) && !loading ? (
         children
       ) : (
-        <StyledLoading type={type} dataTest={dataTest} id={id}>
+        <div
+          className={cx([
+            "flex",
+            "items-center",
+            "justify-center",
+            "h-[120px]",
```

Comment:
it will be a default and if type has it's own height -> it will override that value above, but agree that better to be explicit here and define for each type.